src/megaclient.cpp

```diff
@@ -6762,14 +6762,49 @@ int MegaClient::inited25519()
     {
         free(pubKey);
         LOG_err << "Error deriving the Ed25519 public key.";
-        return(0);
+        return 0;
     }
 
-    // Store the key pair to user attributes.
-    putua("prEd255", (const byte*)signkey.keySeed, crypto_sign_SEEDBYTES, 1);
-    putua("puEd255", (const byte*)pubKey, crypto_sign_PUBLICKEYBYTES, 0);
+    // Store the key pair to user attributes (skipping the procresult())
+    int creqtag = reqtag;
+    reqtag = 0;
+    putua("*prEd255", (const byte*)signkey.keySeed, crypto_sign_SEEDBYTES);
```

Comment:
This is the one that will go terribly wrong. If it's a private attribute (and prefixed with `*`), it *MUST* be TLV-encoded before storage, as it can contain a (hash) map of multiple elements.